test/integration/test_integration.py

```diff
@@ -1278,6 +1278,32 @@ def test_autoquant_compile(self, device, dtype, m1, m2, k, n):
         sqnr = SQNR(out, out2)
         self.assertTrue(sqnr >= 30)
 
+    @parameterized.expand(COMMON_DEVICE_DTYPE)
+    @unittest.skipIf(not TORCH_VERSION_AT_LEAST_2_5, "autoquant requires 2.5+.")
```

Comment:
according to CI, probably need to add skip if cuda is not available, or change this https://github.com/pytorch/ao/blob/629aee11a37d3b79f8352a247f39d35491da3464/test/integration/test_integration.py#L97 to not include cuda when it's not available